setup.py

```diff
@@ -1,35 +1,12 @@
-import sys
-
-if sys.version_info < (2, 6):
-    print >> sys.stderr, "ERROR: galaxy_sequence_utils requires python 2.6 or greater"
-    sys.exit()
-
-# Automatically download setuptools if not available
-from distribute_setup import use_setuptools
-use_setuptools()
-
 from setuptools import setup, find_packages
```

Comment:
Probably safe to assume setuptools is there. bx-python attempts to acquire it using `ez_setup` if not, although the chances of having `ez_setup` and not `setuptools` is probably low?